test/dlc_tests/ec2/tensorflow/training/test_tensorflow_training.py

```diff
@@ -18,13 +18,13 @@
 TF_ADDONS_CMD = os.path.join(CONTAINER_TESTS_PREFIX, "testTFAddons")
 TF_DATASERVICE_TEST_CMD = os.path.join(CONTAINER_TESTS_PREFIX, "testDataservice")
 
-TF_EC2_GPU_INSTANCE_TYPE = get_ec2_instance_type(default="p2.8xlarge", processor="gpu")
+TF_EC2_GPU_INSTANCE_TYPE = get_ec2_instance_type("g3.16xlarge", "gpu")
```

Comment:
why remove keywords? it's nice for clarity imo

Reply:
Reverted this change.